lib/sync-DataSetModel.js

```diff
@@ -489,7 +489,7 @@ var self = {
                 if (lastAccessed + syncClientTimeout < now) {
                   syncUtil.doLog(dataset_id, 'info', 'Deactivating sync for client ' + datasetClient.id + '. No client instances have accessed in ' + syncClientTimeout + 'ms');
                   datasetClient.syncActive = false;
-                  delete self.datasetClientRecords[datasetClient.id];
+                  // delete self.datasetClientRecords[datasetClient.id];
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)